extmod/btstack/modbluetooth_btstack.c

```diff
@@ -75,6 +75,7 @@ STATIC int btstack_error_to_errno(int err) {
 #if MICROPY_PY_BLUETOOTH_ENABLE_CENTRAL_MODE
 STATIC mp_obj_bluetooth_uuid_t create_mp_uuid(uint16_t uuid16, const uint8_t *uuid128) {
     mp_obj_bluetooth_uuid_t result;
+    result.base.type = &mp_type_bluetooth_uuid;
```

Comment:
is this (and the corresponding one for nimble) fixing a bug, or just for extra safety/consistency?

Reply:
Previously, the function that pulled data off the ringbuffer re-created the uuid objects from the raw data and set this base.type then. Now, when not using the ringbuffer, the original object from this function is passed directly to end user irq function so needs this set.

Reply:
I see!